k8s/nginx/staging/global-config.yaml

```
@@ -11,16 +11,144 @@ data:
+          <div class=\"container\">
+              <div class=\"error-code\">401</div>
+              <h1 class=\"title\">Unauthorized</h1>
+              <p class=\"message\">
+                  You need to log in to access this page. Please check your credentials and try again.
+              </p>
+              <a href=\"/\" class=\"home-button\">Back to Home</a>
+          </div>
+      </body>
+      </html>';
+    }
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)